states/huntsman/ready.py

```diff
@@ -9,4 +9,20 @@ def on_enter(event_data):
 
     pocs.observatory.mount.unpark()
```

Comment:
Good to set next state to parking right at the start as a fallback. Lots of states do this.

```suggestion
    pocs.next_state = 'parking'

    pocs.observatory.mount.unpark()
```